policyengine_us/parameters/gov/states/in/tax/income/credits/earned_income/childless/max_age.yaml

```diff
@@ -0,0 +1,9 @@
+description: Indiana has a state EITC that used this maximum age for childless eligiblity.
```

Comment:
```suggestion
description: Indiana limits its earned income tax credit to filers this age or below if they have no children.
```

Reply:
Changed description in both the childless/min_age.yaml and childless/max_age.yaml file.